include/internal/refcount.h

```diff
@@ -15,8 +15,7 @@
 # include <openssl/err.h>
 
 # if defined(OPENSSL_THREADS) && !defined(OPENSSL_DEV_NO_ATOMICS)
-#  if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112L \
-      && !defined(__STDC_NO_ATOMICS__)
+#  if __STDC_VERSION__ >= 201112L && !defined(__STDC_NO_ATOMICS__)
```

Comment:
It's probably fine to do this in internal headers (as long as you're not going to use C++), but it's good that you kept the `#if defined(__STDC_VERSION__)` in the public header e_os.h for that reason.